pilot/pkg/bootstrap/monitoring.go

```diff
@@ -36,13 +38,30 @@ const (
 	versionPath = "/version"
 )
 
+var (
+	uptime      = monitoring.NewGauge("istiod_uptime_seconds", "Current istiod server uptime in seconds")
+	serverStart time.Time
```

Comment:
why not

var serverStart = time.Now()

(or move uptime initialization to init)?

Reply:
it doesn't matter, really (for severStart). i'm happy to move the `time.Now()` to this block.